src/lib/utils/thorchain/lending.ts

```diff
@@ -114,7 +115,11 @@ export const getMaybeThorchainLendingCloseQuote = async ({
   if (!data) return Err('Could not get quote data')
   if ('error' in data) return Err(data.error)
 
-  return Ok(data)
+  return Ok({
+    ...data,
+    // Note, THORCHain is very unlikely to ever return a quote with a memo containing the affiliate name, since you can't have affiliate bps for withdraws
```

Comment:
```suggestion
    // Note, THORCHain is very unlikely to ever return a quote with a memo containing the affiliate name, since you can't have affiliate bps for repayments
```